doc/classes/InputEventJoypadMotion.xml

```diff
@@ -4,6 +4,7 @@
 		Input event type for joypad motion/axis events.
```

Comment:
Input event type for gamepad joysticks and other motions. For buttons see [InputEventJoypadMotion].`